includes/features/stanford_metatag_nobots.feature

```diff
@@ -1,9 +1,10 @@
+@stanford
 Feature: Stanford MetaTag NoBots
   In order to ensure that upgrades do not break existing functionality
   As an administrative user
   I want to ensure that the Stanford MetaTag NoBots module is working properly
 
-  @api
+  @api @safe @deploy
```

Comment:
Here's where we start to run into issues with our definitions. If this test succeeds, it's "safe". If it fails, it leaves the stanford_metatag_nobots module enabled, which could be very bad on a prod website. I would recommend `@api @deploy` for this one.